### PR TITLE
New method: endsWith (90 B)

### PR DESCRIPTION
**Pull request by BabkinAleksandr**
[Ramda's endsWith](http://ramdajs.com/docs/#endsWith)

### CODE REVIEW

File: SIZES.md [hunk 1]
```diff
@@ -76,6 +77,8 @@
 | indexed | 40 B | n/a B | n/a B |
 | indexOf | 110 B | 1570 B | -1460 B |
 | init | 152 B | 493 B | -341 B |
+| insert | 124 B | 378 B | -254 B |
```
Comment (Beraliv) [inline SIZES.md]: what's that? an below as well

File: lib/endsWith/index.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import _curry2 from '../_internal/_curry2'
+
+export default _curry2(function endsWith(suffix, list) {
+  return suffix === list[list.length - 1]
```
Comment (Beraliv) [inline lib/endsWith/index.js]: what if `list` is null or empty?

File: SIZES.md (hunk 1 above)
Comment (BabkinAleksandr) [reply SIZES.md]: PR's of @Kelin2025 has no size calculations

Comment (Beraliv) [reply SIZES.md]: Which one? Please leave a link to it

File: lib/endsWith/index.js (hunk 2 above)
Comment (BabkinAleksandr) [reply lib/endsWith/index.js]: if empty returns false
if null - throws Error

Do not understand whether we using not-throwing-errors politics? I could make checking:

```js
return list ? s === l[l.length-1] : false
```

Comment (Beraliv) [reply lib/endsWith/index.js]: Shorter `return list && s === l[l.length-1]```

File: SIZES.md (hunk 1 above)
Comment (BabkinAleksandr) [reply SIZES.md]: #26, #27, #30, #31

Comment (Beraliv) [reply SIZES.md]: thanks

File: lib/endsWith/index.js (hunk 2 above)
Comment (Beraliv) [reply lib/endsWith/index.js]: fix it for dev mode

**Pull request merged**